### DISCUSSION TITLE
Add custom method to Object with TS in FabricV6

### DISCUSSION THREAD

**Discussion by kiguri** [fabricjs/fabric.js#9118] [Q&A] [answered]
I want to add a custom method to every Fabric Object like this
`fabric.Object.prototype.addLabel = function (label: string) {
  this.set('label', label)
}`

and TS error: `Property 'addLabel' does not exist on type 'FabricObject<any, any, any>'`

How to make it work with TS?

**Comment by ShaMan123** [comment, marked as answer]
This is pure TS
https://www.typescriptlang.org/docs/handbook/declaration-merging.html
